tools/travis/stage.sh

```diff
@@ -23,6 +23,7 @@ STAGE_LIBRARIES="libraries"
 STAGE_CONNECTORS="connectors"
 STAGE_TESTS="tests"
 STAGE_MISC="misc"
+STAGE_PYTHON="python"
```

Comment:
Put this line under `STAGE_CORE="core"` to keep the order same in `.travis.yml`

Reply:
Done.